AliceFaberAcmeDemo/controllers/alice-controller/src/styles.scss

```diff
@@ -1,2 +1,3 @@
 /* You can add global styles to this file, and also import other style files */
-@import "~bootstrap/scss/bootstrap";
\ No newline at end of file
+// @import "~bootstrap/scss/bootstrap";
+@import "../node_modules/bootstrap/scss/bootstrap";
```

Comment:
Is there a cleaner way to import without using relative paths to `node_modules`

Reply:
I see it's done this way even in the bootstrap docs:

https://getbootstrap.com/docs/5.0/customize/sass/#importing